source/classes.tex

```diff
@@ -2075,6 +2076,10 @@
 \pnum
 \indextext{restriction!destructor}%
```

Comment:
@jensmaurer Can we make the indexing similar for constructor and destructor, i.e. have "restriction" and "address of" entries for both?

Reply:
Sure. Done.